src/sidebar/Songs.js

```diff
@@ -20,14 +20,14 @@ export default class Songs extends React.PureComponent {
   renderSong(song) {
     return (
       <div className="song" key={song.id}>
-        <a onClick={() => this.props.changeSong(song.id)}>
-            <img src={`img/album_covers/${song.id}.jpg`} draggable="false" />
+        <button type="button" onClick={() => this.props.changeSong(song.id)}>
+            <img src={`img/album_covers/${song.id}.jpg`} draggable="false" alt="song" />
```

Comment:
Could change this to:

```suggestion
            <img src={`img/album_covers/${song.id}.jpg`} draggable="false" alt={song.name} />
```